perf/benchmark/runner/runner.py

```diff
@@ -481,7 +473,7 @@ def define_bool(parser, opt, help_arg, default_val):
 def main(argv):
     args = get_parser().parse_args(argv)
     print(args)
-    return run(args)
+    return run_perf_test(args)
```

Comment:
we have two run() funcs in this file, to make it much more clear, I renamed this main run perf function to be run_perf_test().